orchagent/bufferorch.cpp

```diff
@@ -61,31 +62,61 @@ void BufferOrch::initTableHandlers()
     m_bufferHandlerMap.insert(buffer_handler_pair(APP_BUFFER_PORT_EGRESS_PROFILE_LIST_NAME, &BufferOrch::processEgressBufferProfileList));
 }
 
-void BufferOrch::initBufferReadyLists(DBConnector *db)
+void BufferOrch::initBufferReadyLists(DBConnector *applDb, DBConnector *confDb)
 {
-    // The motivation of m_ready_list is to get the preconfigured buffer pg and buffer queue items
-    // from the database when system starts.
-    // When a buffer pg or queue item is updated, if the item isn't in the m_ready_list
+    // Map m_ready_list and m_port_ready_list_ref are designed to track whether the ports are "ready" from buffer's POV
```

Comment:
can you change it to a block of comment using /* instead of multiple lines of //?

Reply:
Fixed